doc/non_dev_deployments.md

```diff
@@ -42,9 +42,10 @@ for the public facing endpoints, instead of using self signed certificates.
 
 In that case, the operator must manually upload the certificates:
 
- 1. Store the certificates in the `credentials-high` password store, using
-    this naming convention for the key: `certs/${DEPLOY_ENV}/${CERT_NAME}.crt`
-    and `certs/${DEPLOY_ENV}/${CERT_NAME}.key`.
+ 1. Store the certificates in the `credentials-high` password store, using this
+    naming convention for the key:
+    `certs/${AWS_ACCOUNT}/${DEPLOY_ENV}/${CERT_NAME}.crt` and
+    `certs/${AWS_ACCOUNT}/${DEPLOY_ENV}/${CERT_NAME}.key`.
```

Comment:
It would be nice if these 2 were just below each other in rendered readme like they are here (currently they might end up on a single line, depending on your browser window width).